handbook/benefits.md

```diff
@@ -405,13 +473,206 @@ dbt Labs may, in its sole discretion and unless prohibited by law, require an em
 **How To Request a Leave of Absence:** To help employees navigate their potential options dbt Labs is partnering with [Sparrow](https://trysparrow.com/), a Leave Management Partner. 
 
 Anyone who may need to take a leave of absence should either:
-a) contact their [People Business Partner Team](https://www.notion.so/7c4b9fca56e245e684bd8d8e79f65910) member, or
+a) contact their [People Business Partner Team](https://www.notion.so/dbtlabs/Find-your-PBP-f54e1de2eefa413bad7eb251a3a7c19f?pvs=4) member, or
 b) submit your request via this [**leave request form](https://form.asana.com/?k=KZqlYJHKhsQoYJvtFJXibw&d=312418853397011)** 
 
 The People Support team will introduce you to a Sparrow Leave Specialist as soon as your leave is approved. Sparrow will schedule a call with you to discuss your leave plans, identify the key dates when they need to be involved, and gather the necessary information to file your leave paperwork on your behalf.
 
 Employees who will be going on a leave of absence should review the related policy outlined above.
 
+## [Stipends](https://www.notion.so/dbtlabs/Perks-Stipends-60efa8c883aa4354b361baeefb86c2c9?pvs=4)
+
+As part of dbt’s total rewards strategy, we provide various stipends to help cover some of the cost associated with working from home; as well as supporting our team members' development and well being. These policies apply to all full-time employees that work for any dbt Labs entity or who are employed through our Professional Employer Organizations (PEO - ie, Remote.com).
+
+### [Office Stipend](https://www.notion.so/dbtlabs/Office-Stipend-5f5a37d1026d4fb78ee3c6439fada6dc?pvs=4)
+
+#### Policy
+
+There are many advantages to building a remote-first culture, however we understand that this remote first approach can become a burden on employees that now need to set up a home office that is conducive to their work. This policy is designed to help mitigate some of that burden and pass our savings onto you. 
```

Comment:
The first mention of Benepass is below, explaining the office stipend for new hires. We should introduce the tool up her and link the Benepass website. Maybe put this section (https://www.notion.so/dbtlabs/Perks-Stipends-60efa8c883aa4354b361baeefb86c2c9?pvs=4#0c9ba5d3ac7b4bf2bb4416c52b96c627) here?

Reply:
Actually I see it below - I think that should move up